Browser_IDE/executionEnvironment.html

```diff
@@ -21,6 +21,7 @@
 		</div>
 		<div class="sk-contents">
 			<div id="output" class="sk-contents sk-contents-darker sk-terminal"></div>
+            <input type="text" id="user-input" class="sk-input" style="width:100%; padding:0.5em; margin-top:0.5em;" placeholder="Type command here and press Enter..." onkeydown="handleUserInput(event)">
```

Comment:
No more need for this - handled in #82 which I just merged - just merge with the `latest` main and you'll have it 😃